py/htm/encoders/date.py

```diff
@@ -21,7 +21,7 @@
 import numpy
 import math
 
-from htm.encoders.scalar_encoder import ScalarEncoder, ScalarEncoderParameters
+from htm.bindings.encoders import ScalarEncoder, ScalarEncoderParameters
```

Comment:
we should use the bindings version (ie no pyplot dependency), moved scalar_encoder to examples.

Reply:
the scalar_encoder only imports `pyplot` if you try to use it.

Reply:
true. Is it still ok to use the bindings version, or is there any advantage to using the py wrapper?

Reply:
There is no advantage or disadvantage